src/app/shared/components/template/components/accordion/accordion.component.scss

```diff
@@ -1,11 +1,11 @@
 // Create overlapping effect
 ion-accordion {
-  background: white;
-  border: var(--border-standard);
-  border-radius: 10px;
-  margin-top: -12px;
-  padding-top: 8px;
-  box-shadow: var(--ion-default-box-shadow);
+  background: var(--accordion-background-color, white);
```

Comment:
nit(blocking): when introducing a new component-level theme var such as this, we should use the pattern of extracting that to a SASS `$` variable and defining that at the top of the file, including the fallback value. This makes it easy to see at a glance what values within a component-level stylesheet are exposed to theming. See an example in `packages/components/demo/basic/basic.component.scss`